index.html

```diff
@@ -226,5 +225,46 @@ <h2>About Me</h2>
             </div>
         </section>
 
+        <section class="contact">
+            <div class="contact-container">
+            <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+               <ul>
+            <li><input name="name"  id="name" required maxlength="30" type="text" placeholder="Full name"><hr></li>
+            <li><input name="email-address"  id="email" required type="email" placeholder="Email address"><hr></li>
+            <li><textarea name="message" id="message" minlength="500" placeholder="Enter text here"></textarea></li>
```

Comment:
- [ ] @Team good job so far, kindly check your attribute `minlength` , it supposes to be `maxlength`.Also, make sure your form submits data to your `Formspree` account.